chicken-sloths/bangazon-site-sprint3
# Kb db gen

# Description
- Added `npm run db:generate` to build the database populated by the generated JSON.

## Problem to Solve
Builds a database.

## Steps to Test Solution
1. `psql`
1. `create database sprint3;`
1. exit `psql`
1. change `app/models/config/config.json` to match your default psql credentials
1. `npm run db:gen`
1. Use SQLectron or whatever to check that the stuff is in the DB.
